pipelines/ocw/docker-image-pipeline.yml

```diff
@@ -1,37 +0,0 @@
----
-resources:
-- name: ol-infrastructure
-  type: git
-  source:
-    uri: https://github.com/mitodl/ol-infrastructure.git
-    branch: ((ol-infrastructure-git-ref))
-    paths:
-    - dockerfiles/ocw/node-hugo/Dockerfile
```

Comment:
So, this Docker image is actually not a resource, just an image that has all of the pieces needed to be able to run the course builds and asset compilation.